### PR TITLE
Bug 1127776 - Do not treat $ as punctuation character

### PR DESCRIPTION
**Pull request by janjongboom**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[janjongboom Jan Jongboom (janjongboom)](https://github.com/janjongboom) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=325ee0514831)

File: apps/keyboard/test/unit/imes/latin/worker_test.js [hunk 1]
```diff
@@ -496,4 +504,14 @@ suite('Latin en_us worker', function() {
       });
     });
   });
+
+  suite('fr predictions', function() {
+    suiteSetup(function(next) {
+      setupLanguage('fr', qwertyKeymap, next);
```
Comment (mnjul) [inline apps/keyboard/test/unit/imes/latin/worker_test.js]: The (default) French layout is azerty and I suggest we use the correct layout.

**Comment by try-server-hook** [comment]
[janjongboom Jan Jongboom (janjongboom)](https://github.com/janjongboom) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a1e90c7786e4)

**Comment by try-server-hook** [comment]
[janjongboom Jan Jongboom (janjongboom)](https://github.com/janjongboom) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=bd7aec9ac7f6)

**Pull request merged**